src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -40,6 +41,8 @@ public class CommandTestUtil {
     public static final String VALID_TAG_FRIEND = "friend";
     public static final String VALID_BIRTHDAY_AMY = "01/02/2001";
     public static final String VALID_BIRTHDAY_BOB = "02/03/2001";
+    public static final String VALID_MONEYOWED_AMY = "23.40";
+    public static final String VALID_MONEYOWED_BOB = "1.40";
```

Comment:
MONEYOWED should be separated by underscore. I think you can Shift+F6 to rename it and then IntelliJ will rename your variable throughout the whole project.

Reply:
Made necessary changes